examples/bindingdb_deepdta/.gitignore

```diff
@@ -0,0 +1 @@
+tb_logs/*
```

Comment:
We should have ignore in just one under root for compactness.

Reply:
@bobturneruk Did you notice this above comment? I think it is not necessary to create separate `gitignore`. Is there a reason that this needs to be here rather than in the root?

Reply:
I'll fix.

Reply:
Fixed.